ambari-server/src/main/resources/custom_actions/scripts/stack_select_set_all.py

```diff
@@ -65,7 +65,7 @@ def actionexecute(self, env):
     # this script runs on all hosts; if this host doesn't have stack components,
     # then don't invoke the stack tool
     # (no need to log that it's skipped - the function will do that)
-    if is_host_skippable(stack_selector_path):
+    if is_host_skippable(stack_selector_path, summary.associated_version):
```

Comment:
It's amazing - we got by without this for years, then we add it and boom - something else needs it :)